test/intl402/Locale/constructor-parse-twice.js

```diff
@@ -13,77 +13,35 @@ info: |
   30. Let r be ! ApplyUnicodeExtensionToTag(tag, opt, relevantExtensionKeys).
 
   ApplyOptionsToTag( tag, options )
-  9. If tag matches neither the privateuse nor the grandfathered production, then
```

Comment:
```
ApplyOptionsToTag( tag, options )

1. Assert: Type(tag) is String.
2. If IsStructurallyValidLanguageTag(tag) is false, throw a RangeError exception.
```

is probably the most fundamental piece in these descriptions to make a reference for the RangeError assertions